### PR TITLE
techdebt: [PIE-6636]: Revert PIE-6499

### PR DESCRIPTION
**Pull request by prashantsharma04**
## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
    - CodeFormat: `trigger codeformat`
    - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/40666)

### CODE REVIEW

File: pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java [hunk 1]
```diff
@@ -288,7 +285,7 @@ private void sendUpdateEventIfAny(OrchestrationGraph orchestrationGraph) {
   public OrchestrationGraph buildOrchestrationGraph(String planExecutionId) {
     log.warn(String.format(
         "[GRAPH_ERROR]: Trying to build orchestration graph from scratch for planExecutionId [%s]", planExecutionId));
-    PlanExecution planExecution = planExecutionService.getPlanExecutionMetadata(planExecutionId);
+    PlanExecution planExecution = planExecutionService.get(planExecutionId);
```
Comment (archit-harness) [inline pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java:288]: can be kept same

File: pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java [hunk 2]
```diff
@@ -372,6 +372,9 @@ public NodeExecution update(
   @VisibleForTesting
   @Override
   public boolean shouldLog(Update updateOps) {
+    if (!orchestrationLogConfiguration.isReduceOrchestrationLog()) {
```
Comment (archit-harness) [inline pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java:375]: ignore this

File: pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java [hunk 3]
```diff
@@ -480,7 +485,9 @@ public NodeExecution updateStatusWithUpdate(@NotNull String nodeExecutionId, @No
         if (updated.getStepType().getStepCategory() == StepCategory.STAGE || StatusUtils.isFinalStatus(status)) {
           emitEvent(updated, OrchestrationEventType.NODE_EXECUTION_STATUS_UPDATE);
         }
-        orchestrationLogPublisher.onNodeStatusUpdate(NodeUpdateInfo.builder().nodeExecution(updated).build());
+        if (orchestrationLogConfiguration.isReduceOrchestrationLog()) {
```
Comment (archit-harness) [inline pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java:488]: ignore this complete file

File: pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/plan/PlanExecutionServiceImpl.java [hunk 4]
```diff
@@ -106,7 +108,8 @@ public PlanExecution updateStatusForceful(
     if (ops != null) {
       ops.accept(updateOps);
     }
-    PlanExecution updated = planExecutionRepository.updatePlanExecution(query, updateOps, false);
+    PlanExecution updated = mongoTemplate.findAndModify(
```
Comment (archit-harness) [inline pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/plan/PlanExecutionServiceImpl.java:111]: ignore this change

**Comment by prashantsharma04** [comment]
trigger messagecheck

**Review by archit-harness — APPROVED** [review]
> Reviewed 6 of 16 files at r1, 9 of 10 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/40666)* status: 16 file(s) to review. 1 of 2 approvals obtained. 

___
*[`pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java` line 288 at r1](https://reviewable.io/reviews/harness/harness-core/40666#-NHialVA8IYENlCYi53G-r1-288:-NHwYiGcE30lpAVpWStU:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/7ac2e92bccde5400bfb5d2623c2b1c25de05218d/pipeline-service/modules/orchestration-visualization/src/main/java/io/harness/service/impl/GraphGenerationServiceImpl.java#L288)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

can be kept same
</blockquote></details>

done
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java` line 375 at r1](https://reviewable.io/reviews/harness/harness-core/40666#-NHialVA8IYENlCYi53H-r1-375:-NHwYsOm3epqmqwSShCD:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/7ac2e92bccde5400bfb5d2623c2b1c25de05218d/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java#L375)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

ignore this
</blockquote></details>

done
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java` line 488 at r1](https://reviewable.io/reviews/harness/harness-core/40666#-NHialVA8IYENlCYi53H-r1-488:-NHwYxSa1_mkqaGX4LYg:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/7ac2e92bccde5400bfb5d2623c2b1c25de05218d/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/node/NodeExecutionServiceImpl.java#L488)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

ignore this complete file
</blockquote></details>

done
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/plan/PlanExecutionService.java` line 47 at r1](https://reviewable.io/reviews/harness/harness-core/40666#-NHialVA8IYENlCYi53I-r1-47:-NHwYyPH1RCl4ygCIrXP:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/7ac2e92bccde5400bfb5d2623c2b1c25de05218d/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/plan/PlanExecutionService.java#L47)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

ignore this change
</blockquote></details>

done
___
*[`pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/plan/PlanExecutionServiceImpl.java` line 111 at r1](https://reviewable.io/reviews/harness/harness-core/40666#-NHialVA8IYENlCYi53J-r1-111:-NHwYzCS8UcsBv_aqZFU:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/7ac2e92bccde5400bfb5d2623c2b1c25de05218d/pipeline-service/modules/orchestration/src/main/java/io/harness/engine/executions/plan/PlanExecutionServiceImpl.java#L111)):*
<details><summary><i>Previously, archit-harness (Archit Singla) wrote…</i></summary><blockquote>

ignore this change
</blockquote></details>

done

**Review by naidu-harness — APPROVED** [review]
> Reviewed 6 of 16 files at r1, 3 of 10 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/40666)* status: 6 file(s) to review.

**Pull request merged**